src/remixAppManager.js

```diff
@@ -12,7 +12,7 @@ const requiredModules = [ // services + layout views + system views
 const settings = {
   permissionHandler: new PermissionHandler(),
   autoActivate: false,
-  natives: ['vyper', 'workshops', 'ethdoc', 'etherscan'] // Force iframe plugin to be seen as native
+  natives: ['vyper', 'lexon', 'workshops', 'ethdoc', 'etherscan'] // Force iframe plugin to be seen as native
```

Comment:
this array contains plugins we (remix team) are handling, so i m afraid we can't put `lexon` in here.

Reply:
oh ok. I thought because vyper was there it might be necessary for passing compilation.